tests/system/feature_store/test_feature_store.py

```diff
@@ -82,6 +82,18 @@ def _ingest_stocks_featureset(self):
         stocks_set = fs.FeatureSet(
             "stocks", entities=[Entity("ticker", ValueType.STRING)]
         )
+
+        try:
+            df = fs.ingest(
+                stocks_set,
+                stocks,
+                infer_options=fs.InferOptions.default(),
+                run_config=fs.RunConfig(local=True),
+            )
+            assert False
+        except mlrun.errors.MLRunInvalidArgumentError:
+            pass
```

Comment:
The way to assert something is failed is `with pytest.raises(...` we use it a lot, search the code base